demo/app/lib/main.dart

```diff
@@ -65,60 +94,49 @@ class _MyStatefulWidgetState extends State<MyStatefulWidget> {
-                    borderRadius: BorderRadius.all(
-                      Radius.circular(10.0),
-                    ),
-                    borderSide: BorderSide(
-                      width: 0,
-                      style: BorderStyle.none,
-                    ),
-                  ),
-                  labelText: 'Enter Username',
-                ),
-              ),
+              padding: const EdgeInsets.fromLTRB(0, 0, 0, 24),
+              child: const Text('Trustbloc Sign In',
+                  textAlign: TextAlign.center,
+                style: TextStyle(fontWeight: FontWeight.bold, fontSize: 24, fontFamily: 'SF Pro' )),
             ),
-            SizedBox(height: size.height * 0.02),
             Container(
-                height: 50,
-                padding: const EdgeInsets.fromLTRB(10, 0, 10, 0),
-                child: ElevatedButton(
-                  style: ElevatedButton.styleFrom(
-                    backgroundColor: Colors.white,
-                    shape: RoundedRectangleBorder(
-                      borderRadius: BorderRadius.circular(12), // <-- Radius
-                    ),
-                  ),
-                onPressed: () async {
-                 final SharedPreferences pref = await prefs;
-                 String? userLoggedIn =  pref.getString("userLoggedIn");
+              padding: const EdgeInsets.fromLTRB(24, 0, 24, 0),
+              child: PrimaryInputField(textController: _usernameController, titleTextAlign: TextAlign.center, labelText: 'Username',),
+            ),
+              Container(
+                padding: const EdgeInsets.fromLTRB(24, 24, 24, 0),
+                child:   PrimaryButton(
+                  width: double.infinity,
+                  onPressed: () async {
+                    final SharedPreferences pref = await prefs;
+                    String? userLoggedIn =  pref.getString("userLoggedIn");
                     if (_usernameController.text == userLoggedIn.toString()){
                       _navigateToDashboard();
                     } else {
-                       await _createDid();
-                       pref.setString('userLoggedIn', _usernameController.text);
-                       pref.setString('userDID', userDIDId);
+                      await _createDid();
+                      pref.setString('userLoggedIn', _usernameController.text);
+                      pref.setString('userDID', userDIDId);
                       _navigateToDashboard();
                     }
                   },
-                  child: const Text('Register', style: TextStyle(fontSize: 22, color: Colors.deepPurple)),
-                )),
+                  child: const Text('Sign In ', style: TextStyle(fontSize: 16, color: Colors.white)),
+                ),
+              ),
+            Container(
+              padding: const EdgeInsets.fromLTRB(24, 24, 24, 0),
+              child: const Text('This is a reference app and not to be used for production use cases',
```

Comment:
Can we add a period at the end of this statement ?

Reply:
updated @rolsonquadras